zerver/lib/email_mirror.py

```diff
@@ -107,52 +103,33 @@ def get_missed_message_token_from_address(address: str) -> str:
     if not is_mm_32_format(msg_string):
         raise ZulipEmailForwardError('Could not parse missed message address')
 
-    # strip off the 'mm' before returning the redis key
-    return msg_string[2:]
+    return msg_string
+
+def get_missed_message_address(address: str) -> MissedMessageEmailAddress:
+    token = get_missed_message_token_from_address(address)
+    try:
+        return MissedMessageEmailAddress.objects.select_related().get(
+            email_token=token,
+            used=False,
```

Comment:
I guess with the new `used` model we'd remove this from the filter and just check the value shortly after to determine if it's out of uses.

Reply:
Changed (details in a comment further down)